DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadDocumentManager.cs

```diff
@@ -0,0 +1,148 @@
+using System.Diagnostics;
+using System.Text;
+using Autodesk.AutoCAD.DatabaseServices;
+
+namespace Speckle.Connectors.Autocad.HostApp;
+
+public class AutocadDocumentManager
+{
+  private const string SPECKLE_KEY = "Speckle_DUI3";
+  private const string SPECKLE_MODEL_CARDS_KEY = "Speckle_DUI3_Model_Cards";
+
+  /// <summary>
+  /// Returns all the speckle model cards present in the current document.
+  /// </summary>
+  /// <param name="doc"></param>
+  /// <returns></returns>
+  public string? ReadModelCards(Document doc)
+  {
+    if (doc == null)
+    {
+      return null;
+    }
+
+    using (TransactionContext.StartTransaction(doc))
```

Comment:
We really need to get that ConversionContext through so we can remove these statics

Reply:
This will perhaps require some thought, because I don't see the context in which this AutocadDocumentManager is being used. It does not implement an interface and isn't being injected so we need to consider the relationship with lifetimes - it may be we require named lifetimes...